app/controllers/favorites_controller.rb

```diff
@@ -0,0 +1,73 @@
+require 'tunestakeout.rb'
+
+class FavoritesController < ApplicationController
+
+  def favorite_a_suggestion
+
+    favorite = TunesTakeOut.new.favorite_a_suggestion(params[:user_id],params[:suggestion_id])
```

Comment:
why is this variable called "favorite"? It seems to be a reference to the response from an HTTP request. If so, something like "favorite_response" makes more sense.